test/opentuna.test.ts

```diff
@@ -435,84 +437,6 @@ describe('Tuna Manager stack', () => {
     });
   });
 
-  test('cloudfront distribution without iam cert in China', () => {
```

Comment:
So will we always require the `iamCert` when deploying in China?

Might we convert this test case to validate it.